.github/workflows/label-kernel.yml

```diff
@@ -22,9 +29,23 @@ jobs:
         env:
           CI_EVENT_LABEL_NAME: ${{ github.event.label.name }}
         run: |
-          echo "$CI_EVENT_LABEL_NAME" | sed -n 's/ci:kernel:\([^:]*\):\([^:]*\):*\([^:]*\)$/target=\1/p' | tee --append $GITHUB_OUTPUT
-          echo "$CI_EVENT_LABEL_NAME" | sed -n 's/ci:kernel:\([^:]*\):\([^:]*\):*\([^:]*\)$/subtarget=\2/p' | tee --append $GITHUB_OUTPUT
-          echo "$CI_EVENT_LABEL_NAME" | sed -n 's/ci:kernel:\([^:]*\):\([^:]*\):*\([^:]*\)$/testing=\3/p' | tee --append $GITHUB_OUTPUT
+          if [ -n "${{ inputs.target}}" ]; then
+            echo "target=${{ inputs.target }}"
+          else
+            echo "$CI_EVENT_LABEL_NAME" | sed -n 's/ci:kernel:\([^:]*\):\([^:]*\):*\([^:]*\)$/target=\1/p'
+          fi 2>&1 | tee --append $GITHUB_OUTPUT
+
+          if [ -n "${{ inputs.subtarget}}" ]; then
+            echo "subtarget=${{ inputs.subtarget }}"
+          else
+            echo "$CI_EVENT_LABEL_NAME" | sed -n 's/ci:kernel:\([^:]*\):\([^:]*\):*\([^:]*\)$/subtarget=\2/p'
+          fi 2>&1 | tee --append $GITHUB_OUTPUT
+
+          if [ -n "${{ inputs.testing}}" ]; then
+            echo "testing=${{ inputs.testing }}"
+          else
+            echo "$CI_EVENT_LABEL_NAME" | sed -n 's/ci:kernel:\([^:]*\):\([^:]*\):*\([^:]*\)$/testing=\3/p'
+          fi 2>&1 | tee --append $GITHUB_OUTPUT
```

Comment:
This is totally code taste but isn't it more readable if we first set the target variable and only at the end we export them to GITHUB_OUTPUT without the `| tee --append` ?

Similar to https://github.com/openwrt/actions-shared-workflows/blob/main/.github/workflows/reusable_build.yml#L104